test/functional/pdo_sqlsrv/pdo_065_construct_prefetch.phpt

```diff
@@ -3,29 +3,36 @@ Test unsupported connection attribute ATTR_PREFETCH
 --DESCRIPTION--
 Exception is thrown for the unsupported connection attribute ATTR_PREFETCH only if it is set after PDO::ERRMODE_EXCEPTION is turned on
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
 require_once("MsSetup.inc");
```

Comment:
Please read my comments about the very similar test pdo_065* using ATTR_PERSISTENT

Reply:
oh this test! With column encryption enabled it actually did change the behavior of the test. Without AE, exception's not thrown if PREFETCH is set before ERRMODE, but with AE, exception is actually thrown. I've already made a task for it in the backlog. Anyway, for the same reason I won't remove the enable AE part.

Reply:
Ok then. Please put a comment or TODO here about the pending investigation